tools/scaffolding/templates/gateway/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM node:14-buster as nodebuild
+COPY ./frontend ./frontend
+COPY Makefile .
+RUN yarn --cwd frontend install
+
```

`RUN make frontend` appears to be missing here
```suggestion

RUN make frontend

```

---

Yes indeed it was missing in my local build as well. I'll add it :+1:

---

@daviddyball did you manage to get a working Dockerfile for the scaffolded `clutch-gateway`? I've just started working on this myself using your preliminary template as a starting point. Thanks for sharing your work on it by the way!

---

I've actually not delved into this since I first raised the PR :flushed: If you've got something working then it should be fine to close this PR.

---

Ah ok, thanks for the update! I'm running into some issues executing `RUN make frontend`, so will continue to debug on my end. I'll report back once I've got it sorted.